docs/sphinx/users/comlinetools/formatlist.txt

```diff
@@ -2,34 +2,35 @@ List supported file formats
 ===========================
 
 A detailed list of supported formats can be displayed using the
-:command:`formatlist` command.
+:program:`formatlist` command.
 
-Current usage information can be shown by running:
+.. program:: formatlist
 
-::
+The default behavior is to print a plain-text list of formats::
 
-    formatlist -help
+    formatlist
 
-The default behavior is to print a plain-text list of formats:
+.. option:: -txt
 
-::
+    This can also be accomplished using by specifying the :option:`-txt`
+    argument::
 
-    formatlist
+        formatlist -txt
 
-This can also be accomplished using by specifying the ``-txt`` argument:
+.. option:: -html
 
-::
+    This option allows to print the list of formats as HTML::
```

Comment:
Simply "prints" rather than "allows to print"?

Though other pages seem to use the future tense for some reason, so "This option will print the ..." might also be an option.